readme.md

```diff
@@ -25,6 +25,9 @@ prettyBytes(1337);
 
 prettyBytes(100);
 //=> '100 B'
+
+prettyBytes(42, {signed: true});
```

Comment:
I think it would be good with a short explanation of what this is for. Like when it could be useful. Below the code code block.

Reply:
Good point, I'll add that